### PR TITLE
Add json ouput to crc console command

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: cmd/crc/cmd/console.go [hunk 1]
```diff
@@ -31,45 +33,94 @@ var consoleCmd = &cobra.Command{
...
+		ClusterConfig:           toConsoleClusterConfig(&result),
+		Error:                   errorMessage(err),
+		ConsolePrintURL:         consolePrintURL,
+		ConsolePrintCredentials: consolePrintCredentials,
+	}, writer, outputFormat)
+}
+
+type consoleResult struct {
+	Success                 bool           `json:"success"`
+	State                   state.State    `json:"state"`
```
Comment (guillaumerose) [inline cmd/crc/cmd/console.go:67]: This is redundant with `crc status`. Keep it private ?

File: cmd/crc/cmd/console.go [hunk 2]
```diff
@@ -31,45 +33,94 @@ var consoleCmd = &cobra.Command{
...
+	Error                   string         `json:"error,omitempty"`
+	ClusterConfig           *clusterConfig `json:"clusterConfig,omitempty"`
+	ConsolePrintURL         bool           `json:"consolePrintUrl"`
```
Comment (guillaumerose) [inline cmd/crc/cmd/console.go:70]: This is an implementation details. I don't think we need it in the json ouput. Just make it private ?

Comment (praveenkumar) [reply cmd/crc/cmd/console.go:70]: sure we can make those private.

File: cmd/crc/cmd/console.go (hunk 1 above)
Comment (praveenkumar) [reply cmd/crc/cmd/console.go:67]: Ack

**Review by guillaumerose — APPROVED** [review]

**Comment by cfergeau** [comment]
We probably should not add the `clusterConfig` block when the cluster does not exist:
```
$ ./out/linux-amd64/crc delete
$ ./out/linux-amd64/crc console -ojson
{
  "success": false,
  "error": "Machine 'crc' does not exist. Use 'crc start' to create it",
  "clusterConfig": {
    "webConsoleUrl": "",
    "url": "",
    "adminCredentials": {
      "username": "kubeadmin",
      "password": ""
    },
    "developerCredentials": {
      "username": "developer",
      "password": "developer"
    }
  }
}
```

**Pull request merged**